### PR TITLE
SOF-6896: fix pseudopotential source definition

### PR DESCRIPTION
**Pull request by pranabdas**
Make schema definition in consitent with current web-app schema

### CODE REVIEW

File: example/methods_directory/physical/psp.json [hunk 1]
```diff
@@ -1,49 +1,52 @@
...
+    "slug": "pseudopotential",
     "categories": {
         "tier1": "qm",
         "tier2": "wf",
         "type": "psp",
         "subtype": "paw"
     },
     "data": {
-        "pseudo": [
+        "element": "Ru",
```
Comment (timurbazhirov) [inline example/methods_directory/physical/psp.json]: "data" has to contain an array. We can replace "pseudo" with "metaProperties".

File: example/methods_directory/physical/psp.json [hunk 2]
```diff
@@ -1,49 +1,52 @@
...
-    }
+        ],
+        "apps": ["espresso"],
+        "exchangeCorrelation": { "functional": "pbe", "approximation": "gga" },
+        "name": "pseudopotential",
+        "source": "dojo-oncv",
+        "type": "nc",
+        "version": "0.4"
+    },
+    "source": { "info": {}, "type": "exabyte" }
```
Comment (timurbazhirov) [inline example/methods_directory/physical/psp.json]: This source doesn't make sense, there's already one on line 47

File: schema/methods_directory/physical/psp.json [hunk 3]
```diff
@@ -23,7 +27,20 @@
...
                 }
             }
+        },
+        "source": {
+            "type": "object",
+            "description": "explains where this came from",
+            "properties": {
+                "info": {
+                    "type": "object",
+                    "description": "currently an empty object"
```
Comment (timurbazhirov) [inline schema/methods_directory/physical/psp.json]: See above

File: example/methods_directory/physical/psp.json (hunk 2 above)
Comment (pranabdas) [reply example/methods_directory/physical/psp.json]: Seems this is not used (`web-app/src/application/imports/properties/server/startup.js`), shall we get rid of it?

File: example/methods_directory/physical/psp.json (hunk 1 above)
Comment (pranabdas) [reply example/methods_directory/physical/psp.json]: Ok, need to refactor in `web-app/src/application/imports/properties/server/startup.js`

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**